### ISSUE TITLE
Regtest: Cannot read property '1008' of undefined

### ISSUE THREAD

**Issue by Kixunil** [janoside/btc-rpc-explorer#319]
**Describe the bug**

After opening regtest instance with SSO an error message with stack trace is shown. This is follow up from #311 

**Environment (please complete the following information):**

 - Bitcoin Core / Node Version [e.g. 0.16.3] 0.20
 - NodeJS Version [e.g. 9.x] 10.24.0
 - Browser [e.g. chrome, safari] Firefox
 - Code Version / Commit b4cfba38c532c2dde444c7bb0597aa7af37fb807
 
**Configuration file content**

Sorry, killed VM, can respawn if required.

**To Reproduce**

1. Install Bitcoin stack from cryptoanarchy debian repository
2. Hook the version of btc-rpc-explorer from the above mentioned commit
3. Open btc-rpc-explorer (SSO)

**Screenshots or Log Output**

```
/usr/lib/btc-rpc-explorer/views/includes/index-network-summary.pug:75
    73| 
    74| +contentSection
  > 75| 	+summaryRow(1 + (hashrate7d ? 1 : 0) + (chainTxStats[7 * targetBlocksPerDay] ? 1 : 0) + (difficultyAdjustmentData ? 1 : 0))
    76| 		if (hashrate7d)
    77| 			+summaryItem("7d Hashrate", "The average hashrate of the global mining network over the last 7 days. A comparison to the 30-day average is included to help gauge the rate of change.")
    78| 				- var hashrate7dayData = utils.formatLargeNumber(hashrate7d, 1);

Cannot read property '1008' of undefined

TypeError: /usr/lib/btc-rpc-explorer/views/includes/index-network-summary.pug:75
    73| 
    74| +contentSection
  > 75| 	+summaryRow(1 + (hashrate7d ? 1 : 0) + (chainTxStats[7 * targetBlocksPerDay] ? 1 : 0) + (difficultyAdjustmentData ? 1 : 0))
    76| 		if (hashrate7d)
    77| 			+summaryItem("7d Hashrate", "The average hashrate of the global mining network over the last 7 days. A comparison to the 30-day average is included to help gauge the rate of change.")
    78| 				- var hashrate7dayData = utils.formatLargeNumber(hashrate7d, 1);

Cannot read property '1008' of undefined
    at block (eval at wrap (/usr/lib/btc-rpc-explorer/node_modules/pug-runtime/wrap.js:6:10), <anonymous>:4896:44)
    at Object.pug_interp (eval at wrap (/usr/lib/btc-rpc-explorer/node_modules/pug-runtime/wrap.js:6:10), <anonymous>:174:10)
    at eval (eval at wrap (/usr/lib/btc-rpc-explorer/node_modules/pug-runtime/wrap.js:6:10), <anonymous>:4736:30)
    at template (eval at wrap (/usr/lib/btc-rpc-explorer/node_modules/pug-runtime/wrap.js:6:10), <anonymous>:7471:7)
    at Object.exports.renderFile (/usr/lib/btc-rpc-explorer/node_modules/pug/lib/index.js:454:38)
    at Object.exports.renderFile (/usr/lib/btc-rpc-explorer/node_modules/pug/lib/index.js:444:21)
    at exports.__express (/usr/lib/btc-rpc-explorer/node_modules/pug/lib/index.js:493:11)
    at View.expressApp.engine (/usr/lib/btc-rpc-explorer/app.js:120:23)
    at View.render (/usr/lib/btc-rpc-explorer/node_modules/express/lib/view.js:135:8)
    at tryRender (/usr/lib/btc-rpc-explorer/node_modules/express/lib/application.js:640:10)
    at Function.render (/usr/lib/btc-rpc-explorer/node_modules/express/lib/application.js:592:3)
    at ServerResponse.render (/usr/lib/btc-rpc-explorer/node_modules/express/lib/response.js:1012:7)
    at router.get.asyncHandler (/usr/lib/btc-rpc-explorer/routes/baseRouter.js:279:7)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

**Comment by janoside** [comment]
I believe this was fixed in d37688b. Happy to re-open if you reproduce with d37688b.

**Comment by Kixunil** [comment]
Tested master, works. Thanks!

Would appreciate release somewhat soon. :)
